hack/build-manifest.sh

```diff
@@ -46,7 +47,7 @@ kubectl kustomize ${MANIFESTS_OUT_DIR}/vm > ${MANIFESTS_OUT_DIR}/vm/deployment.y
 
 if [[ $CLUSTER_PROVIDER == "openshift" ]]; then
     cat manifests/${CLUSTER_PROVIDER}/kepler/01-kepler-install.yaml | sed "s|image:.*|image: $IMAGE_REPO:$IMAGE_TAG|" > ${MANIFESTS_OUT_DIR}/kepler/01-kepler-install.yaml
-else
+elif [[ $RUN_K8S_CI == "true" ]]; then
     sed -i "s|image:.*|image: $IMAGE_REPO:$IMAGE_TAG|" ${MANIFESTS_OUT_DIR}/bm/deployment.yaml
```

Comment:
When we deploy in CI (or local cluster) we need to provide a custom $IMAGE_REPO:$IMAGE_TAG 
Otherwise it will use the default ones, so that it should not really change the image name....

The default values should work fine for not CI environment....
```bash
IMAGE_TAG=${IMAGE_TAG:-devel}
IMAGE_REPO=${IMAGE_REPO:-quay.io/sustainable_computing_io/kepler}
```
Maybe the image tag is wrong?

Reply:
I think that the problem is the image tag should be `latest`
So, just changing that should be enough
```bash
IMAGE_TAG=${IMAGE_TAG:-latest}
```

Reply:
we use image: localhost:5001/kepler:devel 
in the CI (make cluster-sync)

Reply:
Actually I am not finding where we define the new `IMAGE_TAG` and `IMAGE_REPO` env variable
```bash
IMAGE_TAG="devel"
IMAGE_REPO ="localhost:5001/kepler"
```
I should be here:
https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/integration_test.yml#L32


And when it is not CI, it should be the default ones

Reply:
For example, In my local environment I do
```bash
IMAGE_REPO ="localhost:5001/kepler" IMAGE_TAG="devel" make cluster-sync
```

Reply:
Passing the correct repo and tag, we don't need a new variable to say it's CI
Also, because the same approach will be used for testing on a local cluster, not just CI

Reply:
so your suggestion is to update the CI script && Test env setup to force localhost:5001/kepler:devel?

Reply:
Yes, that should be the natural way! We should use the IMAGE_REPO and IMAGE_TAG only to define the kepler image...

Some other projects do in this way.... for example [kubevirt](https://github.com/kubevirt/kubevirt)

Reply:
@marceloamaral update. thanks